clickhouse-native-jdbc/src/main/java/com/github/housepower/jdbc/ClickHouseDriver.java

```diff
@@ -39,16 +44,20 @@ public boolean acceptsURL(String url) throws SQLException {
     }
 
     @Override
-    public ClickHouseConnection connect(String url, Properties properties) throws SQLException {
+    public Connection connect(String url, Properties properties) throws SQLException {
```

Comment:
Seems like a breaking change, but I think it does not matter, we guarantee nothing other than standard JDBC interfaces.